.gitignore

```diff
@@ -7,3 +7,4 @@ build/
 dist/
 .idea
 **/node_modules/
+.vscode/
```

Comment:
I can roll this back if needed -- added it because I'm a VS Code user and had some local configs I wanted to ignore.

Reply:
Not a problem at all, thanks for adding it! 🙂